src/test/java/com/commercetools/emailprocessor/model/ProjectConfigurationTest.java

```diff
@@ -0,0 +1,117 @@
+package com.commercetools.emailprocessor.model;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class ProjectConfigurationTest {
+
+
+    private String projectKey = "anyKey";
+    private String clientId = "anyID";
+    private String clientSecret = "anySecret";
+    private String webhookURL = "anyUrl";
+
+    @Test
+    public void isValid() throws Exception {
```

Comment:
Can you give the test a better name?

Reply:
03c4ea7

Reply:
Can't see it from the commit.

Reply:
4b1eaf7d355a3482e38b1220e6c618fa71a2dd0b